templates/openshift/rbac/role-binding.yaml

```diff
@@ -6,5 +6,5 @@ roleRef:
   name: backupjob
 subjects:
   - kind: ServiceAccount
-    namespace: backup
     name: backupjob
+    namespace: webapp
```

Comment:
is there a reason to use "webapp" here?

Reply:
It needs some namespace, otherwise creating it will fail. I tested this in the webapp namespace and it will access all the other namespaces from there.

Reply:
Can you change it to "default"? I believe the shared aws credentials will be stored in the default namespace.